indico/modules/events/management/templates/settings.html

```diff
@@ -6,6 +6,31 @@
 
 {% block content %}
     {{ template_hook('event-actions', event=event) }}
+    {% set move_request = event.pending_move_request %}
+    {%- if move_request -%}
+        <div class="action-box highlight">
+            <div class="section">
+                <div class="icon icon-bookmark"></div>
+                <div class="text">
+                    <div class="label">{% trans %}Pending event moderation{% endtrans %}</div>
+                    {% trans url=move_request.category, category=move_request.category.title %}
```

Comment:
That does not look like a URL ;)

```suggestion
                    {% trans url=move_request.category.url, category=move_request.category.title %}
```